src/components/data-exploration/differential-expression-tool/DiffExprCompute.jsx

```diff
@@ -170,18 +170,28 @@ const DiffExprCompute = (props) => {
               )
             }
             <Tooltip overlay={(
-              <span>
-                For finding genes that are differentially expressed between two experimental groups. This analysis uses a
-                {' '}
-                <a
-                  href='http://bioconductor.org/books/3.14/OSCA.workflows/segerstolpe-human-pancreas-smart-seq2.html#segerstolpe-comparison'
-                  target='_blank'
-                  rel='noreferrer'
-                >
-                  pseudobulk limma-voom workflow
-                </a>
-                .
-              </span>
+              <>
+                <span>
+                  For finding genes that are differentially expressed between two experimental groups. This analysis uses a
+                  {' '}
+                  <a
+                    href='http://bioconductor.org/books/3.14/OSCA.workflows/segerstolpe-human-pancreas-smart-seq2.html#segerstolpe-comparison'
+                    target='_blank'
+                    rel='noreferrer'
+                  >
+                    pseudobulk limma-voom workflow
+                  </a>
+                  .
+                </span>
+                <br />
+                <span>
+                  For between-group comparisons, genes that are unlikely to be significantly differentially expressed due to very
```

Comment:
doesn't this belong to a different PR?

Reply:
yes, included it accidentaly, you can ignore that